src/RecordManager/Base/Record/Lido.php

```diff
@@ -108,6 +108,20 @@ class Lido extends AbstractRecord
      */
     protected $subjectConceptIDTypes = ['uri', 'url'];
 
+    /**
+     * Repository location types to be filtered.
+     *
+     * @var array
+     */
+    protected $repositoryLocationTypes = ['Current location'];
```

Comment:
Ok, for the one below, but $repositoryLocationTypes is “included types only”, so if I make it empty -> won’t collect anything, and I can't do that here? So it would be better to only define/filter the types in the subclass or check here if either empty or has any matching values?